server/src/test/java/org/opensearch/extensions/ExtensionsOrchestratorTests.java

```diff
@@ -485,6 +485,22 @@ public void testHandleActionListenerOnFailureRequest() throws Exception {
         assertEquals("Test failure", extensionsOrchestrator.listener.getExceptionList().get(0).getMessage());
     }
 
+    public void testLocalNodeResponse() throws Exception {
```

Comment:
Thanks for adding this test!  But I'm thinking it should probably be in the same package as the `LocalNodeResponse` class (`src/test/java/org/opensearch/cluster`) as it's not specific to extensions.

Reply:
Sure ill move this test out